oauth/src/test/java/pl/edu/icm/unity/oauth/as/TokenSignerTest.java

```diff
@@ -0,0 +1,80 @@
+/*
```

Comment:
There are 2 test cases missing: positive for EC and negative for RSA.
Also make sure to expect something from the created object when testing sunny paths (e.g. that credential is returned and proper algo).